pkg/identityserver/config.go

```diff
@@ -67,6 +67,7 @@ type Config struct {
 		SMTP         smtp.Config          `name:"smtp"`
 		Templates    emailTemplatesConfig `name:"templates"`
 	} `name:"email"`
+	GatewaySecretKeyID string `name:"gateway-secret-key-id" description:"ID of the key used to encrypt gateway secrets at rest"`
```

Comment:
This would only be needed for encrypting new keys. If you store the DEK/KEK label with the encrypted data, you can easily rotate keys by changing only the config value for newly encrypted secrets.

Reply:
Yeah this works for decrypting since I use the [KeyID that's stored in the secret](https://github.com/TheThingsNetwork/lorawan-stack/blob/2c21c4f2460dfb0105de655fd74951423dadb447/pkg/identityserver/gateway_registry.go#L401).

But this is still needed for [encryption](https://github.com/TheThingsNetwork/lorawan-stack/blob/2c21c4f2460dfb0105de655fd74951423dadb447/pkg/identityserver/gateway_registry.go#L355) right? There's nowhere else I see where I can get this from.